modin/pandas/base.py

```diff
@@ -3127,6 +3127,56 @@ def __init__(
         ]
         self.__groups = self.__get_groups(*self.resample_args)
 
+    _attributes = [
+        "rule",
+        "axis",
+        "closed",
+        "label",
+        "convention",
+        "kind",
+        "loffset",
+        "base",
+        "on",
+        "level",
+        "origin",
+        "offset",
+    ]
+
+    def __getitem__(self, key):
+        if len(self._dataframe.columns.intersection(key)) != len(key):
+            bad_keys = list(set(key).difference(self._dataframe.columns))
```

Comment:
```suggestion
            missed_keys = list(set(key).difference(self._dataframe.columns))
```

Possible this naming will be better .